### DISCUSSION TITLE
v4:饼图弧形渐变如何实现?

### DISCUSSION THREAD

**Discussion by 15802980231** [antvis/G2#5398] [Q&A]
![11259283-9e7b8e291796fe7b](https://github.com/antvis/G2/assets/117149852/0bf076cc-c855-4167-a1f1-869ce63b193f)
使用registerShape,配置线性渐变无法实现示例的效果

**Comment by hustcc** [comment]
环形渐变，可以参考这个：https://g.antv.vision/zh/docs/api/shape/attrs#%E6%94%BE%E5%B0%84%E7%8A%B6%E7%8E%AF%E5%BD%A2%E6%B8%90%E5%8F%98
